cath_alphaflow/commands/convert_dssp_to_sse_summary.py

```diff
@@ -72,20 +77,47 @@ def get_sse_summary_from_dssp(dssp_path, *, chopping=None) -> SecStrSummary:
     else:
         segment_dssp = dssp_string
 
```

Comment:
might be nicer (easier to reuse/test) if we split the following code into a function like:

```
sse_summary = create_sse_summary_from_dssp_str(segment_dssp)
```